<launch>

  <!-- Run the costmap node -->
  <node name="lumyai_navigation_follower_map" pkg="lumyai_navigation_follower" type="navigation_follower_map" output="screen">
  
    <rosparam file="$(find lumyai_navigation_follower)/config/base_local_planner_params.yaml" command="load" ns="TrajectoryPlannerROS" />
    <rosparam file="$(find lumyai_navigation_follower)/config/costmap_common_params_2.yaml" command="load" ns="global_costmap"/>
	<rosparam file="$(find lumyai_navigation_follower)/config/costmap_common_params_2.yaml" command="load" ns="local_costmap"/>
    <rosparam file="$(find lumyai_navigation_follower)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find lumyai_navigation_follower)/config/global_costmap_params.yaml" command="load" />    
    
  </node>

  <!-- Run the map server -->
  <!--node name="map_server" pkg="map_server" type="map_server" args="$(find lumyai_navigation_follower)/maps/map_japan_main_field.yaml"/-->
 
  <!--- Run AMCL -->
  <!--include file="$(find lumyai_navigation_follower)/launch/amcl_node.xml" /-->

    <!--- Run SLAM gmapping -->
  <include file="$(find lumyai_navigation_follower)/launch/slam_gmapping.xml" />
    <!--- Run rviz -->
  <include file="$(find lumyai_navigation_follower)/launch/rviz_move_base.xml" />
</launch>
